elkserver/docker/redelk-base/redelkinstalldata/scripts/modules/alarm_lastline/module.py

```diff
@@ -23,79 +23,33 @@ def __init__(self):
         pass
 
     def run(self):
-        ret = {}
-        alarmLines = []
-        results = {}
+        ret = initial_alarm_result
+        ret['info'] = info
+        ret['fields'] = ['source.ip', 'source.nat.ip', 'source.geo.country_name', 'source.as.organization.name', 'redir.frontend.name', 'redir.backend.name', 'infra.attack_scenario', 'tags', 'redir.timestamp']
+        ret['groupby'] = ['source.ip']
         try:
             report = self.alarm_check1()
-            alarmLines = report.get('alarmLines', [])
-            results = report.get('results', {})
-            # TODO before returning we might have to set an tag on our resultset so we alarm only once. (maybe a tag per alarm?  "ALARMED_%s"%report['fname'] migt do)
-            setTags("ALARMED_%s" % info['submodule'], alarmLines)
+            ret['hits']['hits'] = report['hits']
+            ret['mutations'] = report['mutations'] # for this alarm this is an empty list
+            ret['hits']['total'] = len(report['hits'])
         except Exception as e:
             stackTrace = traceback.format_exc()
             ret['error'] = stackTrace
+            self.logger.exception(e)
```

Comment:
@xychix you need to initialize self.logger in __init__, check in the filehash alarm for example

Reply:
Fixed in #72